2023/typescript/tests/testLib.ts

```
@@ -0,0 +1,5 @@
+import { bench, BenchOptions } from 'vitest'
+
+export const benchmark = (name: string | Function, fn?: () => Promise<any> | any, options?: BenchOptions) => {
```

Biome

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)